test/numerical/TestLSTM.cpp

```diff
@@ -31,7 +30,13 @@ bool isOMLSTMTheSameAsNaiveImplFor(const int direction, const int S,
          lstm.run() && lstm.verifyOutputs();
 }
 
+} // namespace test
+} // namespace onnx_mlir
+
 int main(int argc, char *argv[]) {
+  using namespace onnx_mlir;
+  using namespace onnx_mlir::test;
+
```

Comment:
question for my own understanding: is it a problem if main is inside of the test and onnx_mlir namespace?